tests/manage/test_pvc_concurrent_deletion_creation.py

```diff
@@ -124,7 +123,9 @@ def test_multiple_pvc_concurrent_creation_deletion(self):
         pvc_objs = create_multiple_pvc(self.number_of_pvc, pvc_data)
 
         log.info(f'Created {self.number_of_pvc} new PVCs.')
-        self.pvc_objs_new = pvc_objs[:]
+        global pvc_objs_new
+        pvc_objs_new = [0] * self.number_of_pvc
```

Comment:
@Akarsha-rai can you please explain the purpose of this line?

Reply:
@ebenahar, it will create a list and initialize values to 0, we can remove that.
I'll update in the PR.